tests/functional/c/comparison_of_constants.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=missing-docstring, comparison-with-itself
+
+
+if 2 is 2: # [literal-comparison, comparison-of-constants]
```

Comment:
```suggestion
if 2 is 2:  # [literal-comparison, comparison-of-constants]
```

We add two spaces between a comment and the code. Could you update the other comments accordingly?

Reply:
You can do that by applying black (then ``git add -p``)

Reply:
Note that this file still needs to be updated for spaces.